### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ngrhd**
Added the persistent storage information for users who wish to install custom plugins. This relates to [an old discussion](https://github.com/YOURLS/docker/issues/16#issuecomment-444822154)

### CODE REVIEW

**Comment by yosifkit** [comment]
The content seem fine but I'll let the image maintainers (@LeoColomb, @YOURLS) judge that.

As for the CI failure, that means the change needs to be in respective `content.md` and not `README.md` (it is generated) and that there needs to be no `README` changes in the commits (so a force push of a new commit that just changes `content.md`).

File: yourls/README.md [hunk 1]
```diff
@@ -131,6 +131,17 @@ Database name, defaults to `yourls`. The database must have been created before
 **Optional.**  
 Database tables prefix, defaults to `yourls_`. Only set this when you need to override the default table prefix.
 
+## Persistence Volume
```
Comment (LeoColomb) [inline yourls/README.md:134]: Better if after Docker Secrets

File: yourls/README.md [hunk 2]
```diff
@@ -131,6 +131,17 @@ Database name, defaults to `yourls`. The database must have been created before
...
+
+### `Plugins`
+
+**Optional**  
+If you would like to install custom plugins, you may need to persist the `plugins` folder.
```
Comment (LeoColomb) [inline yourls/README.md:139]:
```suggestion
In order to install additional plugins, `/var/www/html/user/plugins` folder can be mounted as a volume.
```

File: yourls/README.md [hunk 3]
```diff
@@ -131,6 +131,17 @@ Database name, defaults to `yourls`. The database must have been created before
...
+
+```console
+    -v /path/to/your/folder:/var/www/html/user/plugins \
```
Comment (LeoColomb) [inline yourls/README.md:142]:
```suggestion
$ docker run --name some-%%REPO%% -v /path/to/your/folder:/var/www/html/user/plugins ... -d %%IMAGE%%:tag
```

**Review by LeoColomb — COMMENTED** [review]
Good suggestion, @ngrhd!
As mentioned by @yosifkit, only `content.md` should be edited.
Plus see my inline comments.

**Comment by ngrhd** [comment]
Will push a new commit, with necessary changes.

**Comment by LeoColomb** [comment]
Sure! Please ping me on the new one 😃

**Pull request closed without merging**